src/controller/CHIPDeviceController.cpp

```diff
@@ -2442,7 +2482,7 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
     break;
     case CommissioningStage::kFindOperational: {
         // If there is an error, CommissioningStageComplete will be called from OnDeviceConnectionFailureFn.
-        auto scopedPeerId = GetPeerScopedId(proxy->GetDeviceId());
+        auto scopedPeerId = GetPeerScopedId(params.GetRemoteNodeId().Value());
```

Comment:
What guarantees that `GetRemoteNodeId()` has a value here?  Should probably be documented.

Reply:
Add the check for the node ID.